content/getting-started/next.md

```diff
@@ -19,7 +19,7 @@ toc: true
 
 OK so you have [got the jx CLI](/getting-started/install/) and you either
 
-* [created a Kubernetes cluster with Jenkins X](/getting-started/install-on-cluster/)
+* created a Kubernetes cluster with Jenkins X
```

Comment:
Perhaps replace with a correct link instead of remove?

Reply:
Yes, of course.

Somebody should know the correct link.  
I don't know as it is missing in the docs above, the link is actually for existing cluster, not how to get new

Reply:
Should be https://jenkins-x.io/getting-started/create-cluster/.

Reply:
see #1114

Reply:
You have created a new PR. Current should be closed.